src/ModelContextProtocol.Core/Protocol/CreateMessageRequestParams.cs

```diff
@@ -29,13 +29,13 @@ public sealed class CreateMessageRequestParams : RequestParams
     /// response length and computation time. The client may choose to sample fewer tokens than requested.
     /// </remarks>
     [JsonPropertyName("maxTokens")]
-    public int? MaxTokens { get; init; }
+    public required int MaxTokens { get; set; }
```

Comment:
This property was an `int?` despite the spec declaring it as required.

Reply:
We should use the schema as our source of truth when defining these model types.